README.rdoc

```diff
@@ -48,6 +48,10 @@ For initial development, the plugin targets all communication at a vCenter insta
     knife[:vsphere_pass] = "your password"
     knife[:vsphere_dc] = "your-datacenter"
 
+The vSphere password can also be stored in a base64 encoded version (to visually obfuscate it) by prepending 'base64:' to your encoded password. For example:
+
+    knife[:vsphere_pass] = 'base64:VGhhbmtzRXpyYSE=\n'
```

Comment:
Ah, yeah, that is interesting. It seems like the encode method always appends a `\n`, and the decode method intelligently strips it if needed (at least in my irb testing with single and double quotes). 

Regardless, I could drop it from the example if you think it would be less confusing.